src/docs/cookbook/forms/text-field-changes.md

```diff
@@ -131,22 +133,32 @@ Begin listening for changes when the
 `_MyCustomFormState` class is initialized,
 and stop listening when the `_MyCustomFormState` is disposed.
 
-<!-- skip -->
+<?code-excerpt "lib/main.dart (initState)"?>
 ```dart
-class _MyCustomFormState extends State<MyCustomForm> {
-  @override
-  void initState() {
-    super.initState();
+@override
+void initState() {
+  super.initState();
 
-    // Start listening to changes.
-    myController.addListener(_printLatestValue);
-  }
+  // Start listening to changes.
+  myController.addListener(_printLatestValue);
+}
+```
+
+<?code-excerpt "lib/main.dart (dispose)"?>
+```dart
+@override
+void dispose() {
+  // Clean up the controller when the widget is removed from the widget tree.
+  // This also removes the _printLatestValue listener.
+  myController.dispose();
+  super.dispose();
```

Comment:
because the previous text tells to "stop listening when disposed" I added an example of the dispose method here.